src/model.cpp

```diff
@@ -32,6 +32,7 @@
 #include <fstream>
 #include <sstream>
 #include <memory>
+#include <byteswap.h>
```

Comment:
The byteswap.h header does not exist on platform we support (e.g., Windows). We need something more portable, which means either providing manually implemented swaps, or conditionally including platform specific headers.

Reply:
How could i test it on windows ? there are building instructions to build on windows? thanks